### PR TITLE
chore(react-components): add missing exports and change some types

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Chore 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

- Add missing exports
- Rename Image360 add option types
- Allow names from `use3DModelNames` to be undefined (when input models are undefined)

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [ ] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/Image360CollectionContainer/Image360CollectionContainer.tsx [hunk 1]
```diff
@@ -5,18 +5,18 @@ import { type ReactElement, useEffect, useRef } from 'react';
...
+import { type AddImage360CollectionOptions } from '../Reveal3DResources/types';
 import {
   type ImageCollectionModelStyling,
   useApply360AnnotationStyling
 } from './useApply360AnnotationStyling';
 import { type Matrix4 } from 'three';
 
 type Image360CollectionContainerProps = {
-  addImageCollection360Options: AddImageCollection360Options;
+  addImageCollection360Options: AddImage360CollectionOptions;
```
Comment (danpriori) [inline react-components/src/components/Image360CollectionContainer/Image360CollectionContainer.tsx:16]: If it doesnt break anything, maybe it is also good to update the parameter's name to follow the same pattern.

Comment (haakonflatval-cognite) [reply react-components/src/components/Image360CollectionContainer/Image360CollectionContainer.tsx:16]: :+1:

File: react-components/src/components/Reveal3DResources/types.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export type TaggedAdd3DModelOptions = {
 
 export type TaggedAddResourceOptions = TaggedAdd3DModelOptions | TaggedAddImageCollection360Options;
```
Comment (danpriori) [inline react-components/src/components/Reveal3DResources/types.ts:43]: here as well

File: react-components/src/index.ts [hunk 3]
```diff
@@ -99,17 +99,21 @@ export {
...
-  type Image360AssetStylingGroup
-} from './components/Reveal3DResources/types';
-export type {
-  AddImageCollection360Options,
-  AddResourceOptions,
-  AddReveal3DModelOptions
+  type Image360AssetStylingGroup,
+  type TaggedAddResourceOptions,
+  type TaggedAdd3DModelOptions,
+  type TaggedAddImageCollection360Options,
```
Comment (danpriori) [inline react-components/src/index.ts:105]: also here

File: react-components/stories/Image360CollectionContainer.stories.tsx [hunk 4]
```diff
@@ -37,7 +37,7 @@ export const Main: Story = {
 };
 
 type CadModelContainerStoryContentProps = {
-  addImageCollection360Options: AddImageCollection360Options;
+  addImageCollection360Options: AddImage360CollectionOptions;
```
Comment (danpriori) [inline react-components/stories/Image360CollectionContainer.stories.tsx:40]: same here: If it doesnt break anything, maybe it is also good to update the parameter's name to follow the same pattern.

**Review by danpriori — APPROVED** [review]
LGTM. Just some cosmetic suggestions.

**Pull request merged**